src/main/tabs/playing.js

```diff
@@ -492,14 +492,15 @@ Main.k.tabs.playing = function() {
 
 			// Planets
 			if ($("#navModule").length > 0) {
+				var tolltiptext = Main.k.FormatPlanets(0,true);
```

Comment:
Une petite erreur de typo et puis comme les autres variables sont en camel case, il vaudrait mieux :
```suggestion
				var tooltipText = Main.k.FormatPlanets(0,true);
```